content_scripts/link_hints.coffee

```diff
@@ -125,6 +125,22 @@ LinkHints =
     marker
 
   #
+  # Find children elements that have delegated onclick event listener assigned,
+  # and mark them with `vimium-has-delegated-onclick-listener` attribute.
+  # This will make sure those elements are discovered during `getVisibleClickableElements` call.
+  #
+  markChildrenThatHaveDelegatedOnClickListener: (element) ->
+    return unless element.hasAttribute "vimium-jquery-delegated-events-selectors"
+
+    selectorsStr = element.getAttribute "vimium-jquery-delegated-events-selectors"
```

Comment:
This could be rewritten to avoid 2 DOM calls, remove some redundancy:
```
selectorsStr = element.getAttribute "vimium-jquery-delegated-events-selectors"
return unless selectorsStr?
```